libs/deso-protocol/src/lib/identity/Identity.ts

```diff
@@ -418,6 +421,36 @@ export class Identity {
       this.transactions
     );
   }
+  public async messagingGroups(
+    publicKeyBase58Check: string
```

Comment:
we should flesh this out to support all the required parameters. Or make different functions for each - like `registerDefaultKey`, `createMessagingGroup`, and `addMembersToMessagingGroup` or something like that.
1. operation: MessagingGroupOperation,
2. applicationMessagingPublicKeyBase58Check: string,
3. updatedGroupKeyName: string,
4. updatedGroupOwnerPublicKeyBase58Check: string,